test/unit/rpc/getAddress.test.ts

```diff
@@ -10,7 +10,7 @@ describe('Test rpc handler function: getAddress', () => {
 
   const walletStub = new WalletMock();
 
-  afterEach(function () {
+  afterEach(() => {
```

Comment:
Generally in mocha always use function() instead of () =>
Try to switch back to function(){} and see if it works